client/FileUpload.js

```diff
@@ -142,8 +164,10 @@ class FileUpload extends Component {
                     style={{margin: 5}}
                     value={(this.state.fileDoc && this.state.fileDoc[url]) || ""}
                     onChange={this.handleTextChange}
+                    onBlur={this.handleLinkBLur}
```

Comment:
Why onBlue instead of onChange? also typo in function name

Reply:
onChange fires every time you type into the box, so if for some reason a user were to manually type in a link, onBlur makes it so the box waits till they're done before converting the link.
Fixed typo

Reply:
func name still has a capital L but it's whatever

Reply:
It doesn't, the preview on this comment hasn't updated